econml/metalearners.py

```diff
@@ -70,11 +70,10 @@ def fit(self, Y, T, X, inference=None):
         Y, T, X, _ = check_inputs(Y, T, X, multi_output_T=False)
         T = self._label_encoder.fit_transform(T)
         self._one_hot_encoder.fit(T.reshape(-1, 1))
-        self._d_t = len(self._label_encoder.classes_)
-        self._d_y = Y.shape[1:]
-        self.models = check_models(self.models, self._d_t)
+        self._d_t = len(self._label_encoder.classes_) - 1
```

Comment:
To be consistent with other classes, this should probably always be a 1-tuple rather than a raw value. That is,
```suggestion
        self._d_t = len(self._label_encoder.classes_) - 1,
```